### PR TITLE
Adjusting blog brand identity to fit to soliditylang.org

### PR DESCRIPTION
**Pull request by franzihei**
+ Adjusts colors and fonts to follow the same guide as soliditylang.org
+ Moves away from ethereum.org blog branding and makes it clearer this is the Solidity blog. 
+ Adds link to soliditylang.org

Optional: We can also keep using Open Sans as a font for the text bodies, but I'd prefer to make it fit to soliditylang.org. :)

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ share-links-active:
 
 # How to display the link to the website in the footer
 # Remove this if you don't want a link in the footer
-url-pretty: "solidity.ethereum.org"  # eg. "deanattali.com/beautiful-jekyll"
+url-pretty: "soliditylang.org"  # eg. "deanattali.com/beautiful-jekyll"
```
Comment (axic) [inline _config.yml:100]: blog.soliditylang.org

File: _layouts/base.html [hunk 2]
```diff
@@ -19,6 +19,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <!-- Beautiful Jekyll | MIT license | Copyright Dean Attali 2016 -->
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;500;600;700;900&family=Work+Sans:wght@300;400;700&display=swap" rel="stylesheet">
```
Comment (axic) [inline _layouts/base.html:22]: Shouldn't this go into `head.html`?

File: _config.yml (hunk 1 above)
Comment (franzihei) [reply _config.yml:100]: yes, but doesn't work yet. :(

Comment (franzihei) [reply _config.yml:100]: should we leave it as solidity.ethereum.org until then?

File: _layouts/base.html (hunk 2 above)
Comment (franzihei) [reply _layouts/base.html]: I have no idea. Both seems to work so I moved it to head.html!

File: _config.yml (hunk 1 above)
Comment (axic) [reply _config.yml:100]: Well I would rather not change this until it is working :)

**Comment by franzihei** [comment]
Can I merge this now or any other comments? @axic @chriseth

**Comment by axic** [comment]
Curios, do you have a before/after screenshot (especially about the font)?

**Comment by franzihei** [comment]
Yes. 

Before: 
image
After:
image


And in a blog post:

Before: 
image
After: 
image


Actually, I realized that the current version of the blog is inconsistent: The body of blog post previews (on the home page) are Open Sans, while they are Arial if you open the blog post (even though Open Sans is specified). Same happens in the Ethereum blog.

**Comment by axic** [comment]
Tbh I think the new font is harder to read for the blog body, but that is maybe because we are very used to read helvetica/arial everywhere.

**Comment by franzihei** [comment]
You can actually also click on the netlify preview and look at it in your browser, by clicking on the delopy-review details (in the  checks section)!

**Comment by chriseth** [comment]
I'm more concerned about the ragged margin than the font TBH...

**Comment by franzihei** [comment]
But that we currently have in both cases don't we? What do you mean @chriseth? Would you prefer justification?

**Comment by chriseth** [comment]
Yes - not sure how nice browsers now format justified content, but it looks nicer in my opinion...

**Comment by franzihei** [comment]
From a design perspective, I'd say justification is not such a great idea (see some arguments [here](https://designforhackers.com/blog/justify-text-html-css/)).

**Comment by franzihei** [comment]
What can I do to get this merged?

**Review by chriseth — APPROVED** [review]
I actually think the new font is easier to read. The old one was a bit narrow.

**Pull request merged**